pkgs/top-level/all-packages.nix

```diff
@@ -11113,6 +11113,13 @@ in
   julia-stable = julia_15;
```

Comment:
I think we should reshuffle some of these as well. In particular, I don't think it makes sense for any of the "flagship" attributes to point to `julia_15` since that one's broken.

Also I think an alias `julia_16 = julia_16-bin` would make sense.

Reply:
+1 `julia_16 = julia_16-bin`

Reply:
I agree that this makes sense, but I would rather address that in a separate PR and/or issue.